src/graph.rs

```diff
@@ -5005,23 +5302,23 @@ impl<'a, 'graph> Builder<'a, 'graph> {
     }
   }
 
-  #[allow(clippy::too_many_arguments)]
   fn load_pending_module(&mut self, item: PendingModuleLoadItem) {
     let PendingModuleLoadItem {
       redirect_count,
       requested_specifier,
       maybe_attribute_type,
       maybe_range,
       load_specifier,
+      is_asset,
```

Comment:
Just popped into my mind - will url/css/html and any other imports we come up be treated as assets?